src/Blocks/BlockInfested.h

```diff
@@ -0,0 +1,73 @@
+
+// BlockInfested.h
+
+/* This Block Handler describes the blocks spawning silver fishes. Mojang calls them monster egg */
+
+class cBlockInfestedHandler:
+	public cBlockHandler
+{
+	using Super = cBlockHandler;
+
+public:
+
+	using Super::Super;
+
+private:
+
+	virtual cItems ConvertToPickups(NIBBLETYPE a_BlockMeta, const cEntity * a_Digger, const cItem * a_Tool) const override
```

Comment:
Is it actually possible to harvest an infested block?

Reply:
No, not at all.